src/frontend/js/utils/errors/handle.spec.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @jest-environment jsdom
+ */
+import * as Sentry from '@sentry/browser';
+
+jest.mock('@sentry/browser');
+
+describe('handle', () => {
+  (window as any).__richie_frontend_context__ = {
```

Comment:
In the absence of e2e tests, maybe add a test to the backend checking that this is injected to the dom? With a comment in the test to modify frontend before fixing the test?

Reply:
This test already exist in `tests/app/core/test_pages.py`

```py
    @override_settings(SENTRY_DSN="https://example.com/sentry/dsn")
    @override_settings(RELEASE="9.8.7")
    @override_settings(ENVIRONMENT="test_pages")
    def test_page_includes_frontend_context(self):
        """
        Create a page and make sure it includes the frontend context as included
        in `base.html`.
        """
        page = PageFactory(should_publish=True, template="richie/single_column.html")
        response = self.client.get(page.get_public_url())
        self.assertContains(response, '"environment": "test_pages"')
        self.assertContains(response, '"release": "9.8.7"')
        self.assertContains(response, '"sentry_dsn": "https://example.com/sentry/dsn"')

```

Reply:
:+1: Then just add the warning about modifying this test?